### ISSUE TITLE
Composer::installDryRun() calling getProcess without argument

### ISSUE THREAD

**Issue by kbond** [beyondcode/laravel-self-diagnosis#75]
```
Too few arguments to function Illuminate\Support\Composer::getProcess(), 0 passed in /vendor/beyondcode/laravel-self-diagnosis/src/Composer.php on line 9 and exactly 1 expected
```

Looks like Laravel 5.8 changed `Illuminate\Support\Composer::getProcess()` to have a required `array $command` argument.

**Comment by tswestendorp** [comment]
Also ran into this issue, fixed it temporarily using the following code. Not sure if it's compatible Laravel 5.7 and lower. 

```
$process = $this->getProcess([]);

        $process->setCommandLine(trim(implode(' ', $this->findComposer()).' install --dry-run '.$options));
```

**Comment by kbond** [comment]
It is but `Process::setCommandLine()` is deprecated in Symfony 4.2. There is a static constructor for process: `Process::fromShellCommandline()` that accepts a string as the command but it is only available in Symfony 4.2+.

**Comment by fitztrev** [comment]
Having this issue too. It affects these 2 checks, so I've disabled them for now:

```
ComposerWithDevDependenciesIsUpToDate
ComposerWithoutDevDependenciesIsUpToDate
```
